### PR TITLE
Fixes for pocket add page (fixes #12494)

### LINKED ISSUE #12494: Fixes for pocket /add page
## Description

There are a few significant differences between the [marketing /add page](https://www.tekcopteg.com/en/add/) and [pocket prod /add page](https://getpocket.com/add)

---

#### Success Criteria
- [ ] Headline goes full width to avoid overlarge vertical spacing gaps
marketing
prod
- [ ] We confirm the integrated app section links and fix "Integreated" typo (EN only)
marketing-apps
prod-apps
- [ ] Check with Pocket team on altering their CDN routing to use marketing page

### PR DESCRIPTION
**Pull request by maureenlholland**
## One-line summary

Updates to better match the prod /add page

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/12494

## Testing

Run in pocket mode

To test this work:

- [ ] http://localhost:8000/en/add/

top headline should go across section
app integration section
- should have typo fixed
- sets 2 icons per row on larger screens, as we only have 4 working links

### CODE REVIEW

File: media/css/pocket/components/add.scss [hunk 1]
```diff
@@ -213,44 +212,54 @@
     display: flex;
     flex-direction: column;
 
+    h3 + * {
```
Comment (alexgibson) [inline media/css/pocket/components/add.scss:215]: NIt: potentially a broad selector, could this just be `h3 + p`?

**Review by alexgibson — APPROVED** [review]
r+

File: media/css/pocket/components/add.scss (hunk 1 above)
Comment (alexgibson) [reply media/css/pocket/components/add.scss:215]: Actually looking at the DOM more closely, this is probably fine 👍

Comment (maureenlholland) [reply media/css/pocket/components/add.scss]: lol, just made the change. It's probably only ever going to be a `p` following the heading in any case.

**Pull request merged**